tests/lib/tools/suite/journal-state/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: smoke test for the journal-state tool
+execute: |
+    # Check help
```

Comment:
Could we also shellcheck the program here, even if it is also checked by some other layer.

Reply:
done

Reply:
Where is this checked?

Reply:
Sorry, it was included in another pr, now the we run static cehcks through run-checks script all the tools are validated.
Currently the static checks are validated when running the unit/go spread test. Shellcheck package is preinstalled for ubuntu-18.04-64 so we make sure the validation is done.

Reply:
FWIW, journal-state is shellcheck clean atm.